pages/common/bash.md

```diff
@@ -28,6 +28,10 @@
 
 `bash -s`
 
+- Pass arguments behind to the script (can be combined with `-s` to pass arguments to commands/scripts from `stdin`):
+
+`bash {{file.sh}} --`
```

Comment:
In my opinion, `--` disables further options parsing and whatever is given after `-` or `--` is considered as filenames or arguments to the script itself and not to bash.
Could we have a concise example about that.